site/docs/plugins/keyboard.md

```diff
@@ -36,15 +36,11 @@ It provides a class called `InlineKeyboard` for this.
 
 ### Building an Inline Keyboard
 
-Here are three examples how to build an inline keyboard with `text` buttons.
+You can build an inline keyboard by creating a new instance of the `InlineKeyboard` class, and then adding buttons you like to it using `.text()` and its other methods.
 
-You can also use other methods like `url` to let the Telegram clients open a URL, and many more options as listed in the [grammY API Reference](https://deno.land/x/grammy/mod.ts?s=InlineKeyboard#Methods) as well as the [Telegram Bot API Reference](https://core.telegram.org/bots/api#inlinekeyboardbutton) for `InlineKeyboard`.
+Here is an example:
 
-#### Example 1
-
-Buttons for a pagination navigation can be built like this:
-
-##### Code
+![Example 1](/images/inline-keyboard-example.webp)
```

Comment:
Why is the example number here and not [below](https://github.com/grammyjs/website/pull/806/files#diff-3d59b5164dfd33276ca18ca1e8b0bf15b4ec6326c660582c832eeffbebfb6adbR163)?
```suggestion
![Example](/images/inline-keyboard-example.webp)
```

Reply:
Good catch

Reply:
And critical